hello.py

```diff
@@ -20,8 +20,8 @@ def someMethod(args):
                       help='the banana parameter')
   args = parser.parse_args()
   # say hello
-  print('Hello, world')
+  print('Hello, world!')
   # execute a method
   someMethod(args)
   # say goodbye
-  print('Goodnight, world')
+  print('Goodnight, world!')
```

Comment:
We should make the goodnight message end with a period '.' character, since it's bedtime and we don't want to be too loud.